alternate-server-replay.py

```diff
@@ -135,7 +136,8 @@ def next_flow(self, request):
         # if it's an exact match, great!
         if len(flows) == 1:
             candidate = flows[0]
-            if candidate.request.url == request.url and candidate.request.raw_content == request.raw_content:
+            if candidate.request.url == request.url and \
```

Comment:
Interested about this one. I generally prefer to avoid backslash continuations in every case, so I usually prefer adding parens, e.g.

```
  if (candidate.request.url == request.url and 
      candidate.request.raw_content == request.raw_content):
```

Would that solve the linter?